testworld/park.go

```diff
@@ -264,6 +282,42 @@ func (h *host) start(c context.Context) error {
 
 }
 
+func (h *host) kill() {
+	h.canc()
+}
+
+func (h *host) isLive(softTimeOut time.Duration) (bool, error) {
```

Comment:
```suggestion
func (h *host) isLive(timeOut int) (bool, error) {
```

Reply:
Why a soft and a hard time out? Why not just one hard time out passed by the user?

Reply:
something like:
```golang
func (h *host) isLive(timeOut int) (bool, error) {	
sig := make(chan bool)
	c := createInsecureClient()
	go func(sig chan<- bool) {
		var fErr error
		for i := 0; i < timeOut; i++ {
			res, err := c.Get(fmt.Sprintf("https://localhost:%d/ping", h.config.GetServerPort()))
			fErr = err
			if err != nil {
				time.Sleep(time.Second)
				continue
			}
			if res.StatusCode == http.StatusOK {
				sig <- true
				return
			}
		}
		sig <- false
	}(sig)
	return <-sig:
	
```

Reply:
In current implementation the soft timeout is higher than the hard timeout :)

Reply:
Only reason the soft timeout exists is to give the person who writes the test to exit earlier since he doesn't care about host being alive. Right now the only place we use it I have set the soft timeout higher so that hard timeout is used. But someone else can use it in a different way.